xrpl/asyncio/clients/client.py

```diff
@@ -0,0 +1,32 @@
+"""Interface for all network clients to follow."""
```

Comment:
I think this abstract client should be pulled out of the async package and left in the main package - maybe with other objects like `Wallet`